notebooks/external_convergence_shear_distributions_tutorial.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_glass_cls_file_certain_z_sigma8(z, sigma8, Om0, base_file_path):
</pre><p>for this definitions, there is not sufficient documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>